html.go

```diff
@@ -482,8 +482,12 @@ func (options *Html) AutoLink(out *bytes.Buffer, link []byte, kind int) {
 	out.WriteString("</a>")
 }
 
-func (options *Html) CodeSpan(out *bytes.Buffer, text []byte) {
-	out.WriteString("<code>")
+func (options *Html) CodeSpan(out *bytes.Buffer, text []byte, lang string) {
+	if len(lang) > 0 {
```

Comment:
It's more idiomatic to check that a string is not empty by doing `if lang != "" {`.